sv.h

```diff
@@ -422,9 +422,8 @@ perform the upgrade if necessary.  See C<L</svtype>>.
                                            including PVLV-as-regex. See
                                            isREGEXP().
                                        */
-#define SVf_OOK		0x02000000  /* has valid offset value. For a PVHV this
-                                       means that a hv_aux struct is present
-                                       after the main array */
+#define SVf_OOK		0x02000000  /* has valid offset value */
+#define SVphv_HasAUX    SVf_OOK     /* PVHV has an hv_aux struct after the main array */
```

Comment:
I'm not clear on what the "main array" means here. Seems like the `xpvhv_aux` struct is the final member of the `xpvhv_with_aux` struct, which will be allocated from an appropriately-sized xpv arena?

Reply:
I'll be honest - I have no idea myself. I just cloned the wording from the previous version of the comment :) I guess I could just remove the words "after the main array" - it's still clear without that.